scripts/text_classification/exam_readme.md

```diff
@@ -0,0 +1,25 @@
+# AAAI_2019_EXAM
```

Comment:
This should be converted to rst, and linked or inlined in https://github.com/dmlc/gluon-nlp/blob/master/scripts/text_classification/index.rst